go.mod

```diff
@@ -22,3 +24,5 @@ require (
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter v0.0.0 => ./exporter/azuremonitorexporter
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter => ./exporter/sapmexporter
```

Comment:
Can you remove these "replace" directives that are no longer needed?

Reply:
I can but first we need to merge this or we'll need another PR I think. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/59

Reply:
I kind of prefer to go in two PRs but up to you.